models/classes/class.QtiTestService.php

```diff
@@ -873,19 +875,46 @@ private function deleteTestsFromClassByLabel(
         core_kernel_classes_Class $itemClass
     ): void {
         $testService = $this->getTestService();
-        $itemTreeService = $this->getItemTreeService();
+
+        $deletedTests = [];
+        $deletedItemClasses = [];
+        $refs = [];
```

Comment:
maybe it would be nice to type this arrays, i thinkthe first two are string arrays but the third looks like objects :thinking:

Reply:
Type for `$refs` is inferred from the PHPDoc of `collectResourceReferences()`, and the others from the type of data pushed to them. However, I'm adding explicit type annotations in 012a459ce to make it clearer :ok_hand: